<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1460</width>
    <height>818</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>YuMincho</family>
    <pointsize>24</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>571</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>PingFang TC</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>動物醫院資料庫管理系統</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="show_vet_b">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>PingFang TC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>顯示獸醫資料</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="show_dept_b">
        <property name="font">
         <font>
          <family>PingFang TC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>顯示科別資訊</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="show_pat_b">
        <property name="font">
         <font>
          <family>PingFang TC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>顯示病患資料</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="show_owner_b">
        <property name="font">
         <font>
          <family>PingFang TC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>顯示飼主資料</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="in_b">
        <property name="font">
         <font>
          <family>Varela Round</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>IN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="not_in_b">
        <property name="font">
         <font>
          <family>Varela Round</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>NOT IN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exists_b">
        <property name="font">
         <font>
          <family>Varela Round</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>EXISTS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="not_exists_b">
        <property name="font">
         <font>
          <family>Varela Round</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>NOT EXISTS</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="sfw_b">
        <property name="font">
         <font>
          <family>Varela Round</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>SELECT-FROM-WHERE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_b">
        <property name="font">
         <font>
          <family>Varela Round</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>DELETE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="insert_b">
        <property name="font">
         <font>
          <family>Varela Round</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>INSERT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="update_b">
        <property name="font">
         <font>
          <family>Varela Round</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>UPDATE</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="count_b">
        <property name="font">
         <font>
          <family>Varela Round</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>COUNT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sum_b">
        <property name="font">
         <font>
          <family>Varela Round</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>SUM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="max_b">
        <property name="font">
         <font>
          <family>Varela Round</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>MAX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="min_b">
        <property name="font">
         <font>
          <family>Varela Round</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>MIN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="avg_b">
        <property name="font">
         <font>
          <family>Varela Round</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>AVG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="having_b">
        <property name="font">
         <font>
          <family>Varela Round</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>HAVING</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QPushButton" name="show_record_b">
        <property name="font">
         <font>
          <family>PingFang TC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>顯示來診紀錄</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="show_check_b">
        <property name="font">
         <font>
          <family>PingFang TC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>顯示看診紀錄</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>791</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>PingFang TC</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Button內建之SQL指令說明</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="font">
       <font>
        <family>PingFang TC</family>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>378</y>
     <width>571</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>PingFang TC</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>輸入SQL指令</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="enter_sql_cmd">
      <property name="font">
       <font>
        <family>PingFang TC</family>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="enter_b">
        <property name="font">
         <font>
          <family>PingFang TC</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>確認</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>791</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QTableWidget" name="result_display">
      <property name="font">
       <font>
        <family>PingFang TC</family>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>PingFang TC</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>結果顯示</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
